projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py

```diff
@@ -673,6 +674,8 @@ def __helm_install_command(self, git_server_ip):
             "--set",
             "ingress.enabled=true",
             "--set",
```

Comment:
I may have missed it, did we update the docs on this?
VEP-1507 and quickstart-vdk ones

Reply:
reverted.